kubetest/azure.go

```diff
@@ -49,6 +49,7 @@ const (
 	winZipTemplate                = "win-zip-%s.zip"
 	k8sNodeTarballTemplate        = "kubernetes-node-linux-amd64-%s.tar.gz"
 	azureBlobContainerURLTemplate = "https://%s.blob.core.windows.net/%s"
+	azureKubemarkTestPrefix       = "[for azure kubemark test]"
```

Comment:
do we really need a long string here? I think add the word kubemark in the docs is enough

Reply:
OK, just notice it's used for distinguishing from GCE flags, then it makes sense